kotlin/identify-raster-cell/src/main/java/com/esri/arcgisruntime/sample/identifyrastercell/MainActivity.kt

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright 2020 Esri
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.esri.arcgisruntime.sample.identifyrastercell
+
+import android.graphics.Color
+import android.graphics.Point
+import android.os.Bundle
+import android.view.MotionEvent
+import android.widget.TextView
+import androidx.appcompat.app.AppCompatActivity
+import com.esri.arcgisruntime.layers.RasterLayer
+import com.esri.arcgisruntime.mapping.ArcGISMap
+import com.esri.arcgisruntime.mapping.Basemap
+import com.esri.arcgisruntime.mapping.view.DefaultMapViewOnTouchListener
+import com.esri.arcgisruntime.raster.Raster
+import com.esri.arcgisruntime.raster.RasterCell
+import kotlinx.android.synthetic.main.activity_main.*
+
+
+class MainActivity : AppCompatActivity() {
+
+  private var rasterLayer: RasterLayer? = null
+
+  override fun onCreate(savedInstanceState: Bundle?) {
+    super.onCreate(savedInstanceState)
+    setContentView(R.layout.activity_main)
+
+    // load the raster file
+    val rasterFile =
+      Raster(getExternalFilesDir(null)?.path + "/SA_EVI_8Day_03May20/SA_EVI_8Day_03May20.tif")
+
+    // create the layer
+    rasterLayer = RasterLayer(rasterFile)
+
+    // define a new map
+    val rasterMap = ArcGISMap(Basemap.Type.OCEANS, -33.9, 18.6, 9).apply {
+      // add the raster layer
+      operationalLayers.add(rasterLayer)
+    }
+
+    mapView.apply {
+      // add the map to the map view
+      map = rasterMap
+
+      // set behavior for double touch drag and on up gestures
+      onTouchListener = object : DefaultMapViewOnTouchListener(this@MainActivity, mapView) {
+        override fun onDoubleTouchDrag(e: MotionEvent): Boolean {
+          val offset = 200
```

Comment:
Personally I think this offset is too high? It feels like a big jump when I stop dragging and it doesn't feel like it's identifying the cell under my finger. A smaller offset might work better, or if you can remove the pointy bit from the callout?